benchmark/Cpp/Savina/src/parallelism/PiPrecision.lf

```diff
@@ -1,51 +1,25 @@
 /**
- * Parallelism benchmark from the Savina benchmark suite.
- * See https://shamsimam.github.io/papers/2014-agere-savina.pdf.
+ * Copyright (C) 2020 TU Dresden
  * 
+ * This benchmark calculates Pi to a very high precision. The workers operate
+ * independently and each worker calculates one digit of the result. Similar to
+ * to the other parallelism benchmarks, the execution is controlled by a central
+ * manager. The overall structure in this benchmark is quite simple. The manager
+ * tells the workers which digit to calculate and collects the results to build Pi.
+ *
+ * Note that the validation isn't precise. Due to rounding errors on the last digit,
+ * it might return false although the result is correct.
```

Comment:
I assume the same flag exists in the Akka implementation?

Reply:
I guess you mean bug? Yes, I think it has the same problem. Actually, many of the Savina validation routines are bogus. Here, I found it is not worth the effort to fix it.

Reply:
I guess you mean bug? Yes, I think it has the same problem. Actually, many of the Savina validation routines are bogus. Here, I found it is not worth the effort to fix it.